tox.ini

```diff
@@ -24,7 +23,11 @@ deps =
     tensorflow<2.10
 setenv =
     TF_GPU_ALLOCATOR=cuda_malloc_async
+    HOROVOD_GPU_OPERATIONS=NCCL
+allowlist_externals =
+    horovodrun
 commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/nvtabular.git
     python -m pytest --cov-report term --cov merlin -rxs tests/unit
+    horovodrun -np 2 sh examples/usecases/multi-gpu/hvd_wrapper.sh python -m pytest -m horovod -rxs tests/unit
```

Comment:
Would it make sense to add this command to the `Makefile`?

Reply:
Github actions uses the tests in the `Makefile` right? I was focusing on the GPU case so didn't think about adding it to CPU tests. I think we need to add a new workflow to the test matrix because of the horovod dependency so maybe I should follow up with another PR that runs tests on github actions.

Reply:
I was more thinking that it might be useful to be able to trigger the mutli-gpu test locally (using for instance the Makefile) But we can leave that for a future PR as well.